__tests__/checkDateOrder.test.ts

```diff
@@ -1,20 +0,0 @@
-import { expect, test } from '@jest/globals';
```

Comment:
If we think, we need a separate util function for something like this, we should definitely not write it ourselves. Something like lodash should be preferred.